smartsim/_core/entrypoints/colocated.py

```diff
@@ -100,8 +101,9 @@ def launch_db_model(client: Client, db_model: t.List[str]) -> str:
             name=name,
             model_file=args.file,
             backend=args.backend,
-            fist_gpu=0,
+            first_gpu=args.first_device,
             num_gpus=args.devices_per_node,
+            first_gpu=args.first_device,
```

Comment:
Repeated `first_gpu` argument at lines 104 and 106

Reply:
Good catch!